website/www/site/content/en/contribute/release-guide.md

```diff
@@ -1163,7 +1180,7 @@ Use reporter.apache.org to seed the information about the release into future pr
 **********
 
 
-## 9. Promote the release
+## 11. Promote the release
```

Comment:
(Minor question.) Does markdown not support automatic numbering?

Reply:
It does. We use that feature in the table of contents. (You simply always specify `1.` as the number).

I don't know why we don't use it for the actual sections. I guess to keep it plain-text readable. Either way, it would make sense to stick to just one of the two approaches.